templates/overrides/search/facets-summary-item-list.html.twig

```diff
@@ -1,28 +1,14 @@
 {#
 /**
  * @file
- * Default theme implementation for a facets summary item list.
+ * Theme implementation for a facets summary item list in oe_whitelabel.
  *
- * Available variables:
- * - items: A list of items. Each item contains:
- *   - attributes: HTML attributes to be applied to each list item.
- *   - value: The content of the list element.
- * - title: The title of the list.
- * - list_type: The tag for list element ("ul" or "ol").
- * - wrapper_attributes: HTML attributes to be applied to the list wrapper.
- * - attributes: HTML attributes to be applied to the list.
- * - empty: A message to display when there are no items. Allowed value is a
- *   string or render array.
- * - context: A list of contextual data associated with the list. May contain:
- *   - list_style: The custom list style.
+ * @see ./modules/contrib/facets/modules/facets_summary/templates/facets-summary-item-list.html
  *
- * @see facets_summary_preprocess_facets_summary_item_list()
- *
- * @ingroup themeable
  */
 #}
-{%- if items -%}
-  {%- for item in items -%}
+{%- if result_items -%}
+  {%- for item in result_items -%}
     {{ item.value }}
```

Comment:
can't remember why we did `item.value` and not `item`, can you check if it works with the whole render array